tests/test_copy.py

```diff
@@ -0,0 +1,97 @@
+import os
+import sys
+import shutil
+import hashlib
+import unittest
+
+from kapitan.cli import main
+from kapitan.utils import directory_hash
+from kapitan.inputs.copy import Copy
+
+
+search_path = ""
+ref_controller = ""
+test_path = os.path.dirname(os.path.abspath(__file__)) + "/test_copy"
+compile_path = test_path + "/output"
```

Comment:
Same for all these paths. Use `os.path.join`.

Reply:
Changed them with os.path.join.